pages/_app.js

```diff
@@ -1,7 +1,38 @@
+import React from "react"
 import "../styles/globals.css"
 
+// import { Auth0Provider } from "use-auth0-hooks"
+import { Auth0Provider } from "@auth0/auth0-react"
+import Router from "next/router"
+import Navbar from "../components/navbar"
+
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  // console.log(window.location.origin)
+
+  const onRedirectCallback = (appState) => {
+    console.log(appState)
+    Router.push(
+      appState && appState.returnTo
+        ? appState.returnTo
+        : window.location.pathname,
+    )
+  }
```

Comment:
so after successful login, app redirects and loads `/profile` page, 
otherwise URL changes but page with react component does not load (still in /)